### PR TITLE
Add default peer limits configuration

### PR DESCRIPTION
**Pull request by dshulyak**
If discovery is enabled for a given cluster - we will set a default
expected number of peers for each enabled service. For example:

- if cluster is rinkeby has a discovery enabled we will
  check which services are enabled
- if whisper is enabled we will set min and max limits by default
- if les is enabled and infura is not used we will set limits too

When statusd is used - configuration must be provided using configuration
supported by statusd.

### CODE REVIEW

File: geth/params/defaults.go [hunk 1]
```diff
@@ -105,4 +107,12 @@ const (
...
 	// StatusChainNetworkID is id of a test network (private chain)
 	StatusChainNetworkID = 777
+
+	// WhisperDiscv5Topic used to register and search for whisper peers using discovery v5.
+	WhisperDiscv5Topic = discv5.Topic("whisper")
+)
+
+var (
+	// WhisperDiscv5Limits declares min and max limits for peers with whisper topic.
+	WhisperDiscv5Limits = Limits{2, 2}
```
Comment (themue) [inline geth/params/defaults.go:121]: Understand this as lower limit 2 but also upper limit 2, more are not allowed. Am I right?

**Review by themue — APPROVED** [review]
LGTM, only a question.

File: geth/params/defaults.go (hunk 1 above)
Comment (dshulyak) [reply geth/params/defaults.go:121]: right, i think for mobile devices it doesn't make sense to have different upper and lower bounds cause we will stop discovery when max bound is reached anyway

Comment (themue) [reply geth/params/defaults.go:121]: Thanks 👍

**Comment by dshulyak** [comment]
@adambabik can you take a quick look, please? this configuration works for me and @themue, and we can use whisper cluster, but maybe you want to see something else

**Comment by dshulyak** [comment]
Also, i will add support for LES2 topic search but after rebase on 1.8.4

**Review by adambabik — APPROVED** [review]

**Comment by dshulyak** [comment]
i had to add more changes to this PR:

- i found a deadlock in les code, for some reason i was able to reproduce it locally only with -race flag, but it was blocking my CI. But the error is obvious
- i added back 2 static servers with les, otherwise our e2e tests were not passing. I plan to replace them with discovered a bit later

**Review by mandrigin — APPROVED** [review]
Looks good!
And good patch :)

**Pull request merged**